Description: This allows booting an initrd without headers generated
 by mkimage, which is generally needed when using distro_bootcmd with
 extlinux-style boot menus.
Forwarded: https://patchwork.ozlabs.org/patch/1132130/
 https://lists.denx.de/pipermail/u-boot/2019-July/376577.html
Author: Vagrant Cascadian <vagrant@debian.org>

Index: u-boot/configs/qemu-riscv64_defconfig
===================================================================
--- u-boot.orig/configs/qemu-riscv64_defconfig
+++ u-boot/configs/qemu-riscv64_defconfig
@@ -9,3 +9,4 @@ CONFIG_DISPLAY_BOARDINFO=y
 CONFIG_CMD_BOOTEFI_SELFTEST=y
 # CONFIG_CMD_MII is not set
 CONFIG_OF_PRIOR_STAGE=y
+CONFIG_SUPPORT_RAW_INITRD=y
Index: u-boot/configs/qemu-riscv64_smode_defconfig
===================================================================
--- u-boot.orig/configs/qemu-riscv64_smode_defconfig
+++ u-boot/configs/qemu-riscv64_smode_defconfig
@@ -10,3 +10,4 @@ CONFIG_DISPLAY_BOARDINFO=y
 CONFIG_CMD_BOOTEFI_SELFTEST=y
 # CONFIG_CMD_MII is not set
 CONFIG_OF_PRIOR_STAGE=y
+CONFIG_SUPPORT_RAW_INITRD=y
